components/script/dom/element.rs

```diff
@@ -2702,20 +2785,30 @@ impl VirtualMethods for Element {
         }
     }
 
-    fn bind_to_tree(&self, tree_in_doc: bool) {
+    fn bind_to_tree(&self, tree_connected: bool) {
         if let Some(ref s) = self.super_type() {
-            s.bind_to_tree(tree_in_doc);
+            s.bind_to_tree(tree_connected);
         }
 
         if let Some(f) = self.as_maybe_form_control() {
             f.bind_form_control_to_tree();
         }
 
-        if !tree_in_doc {
+        if !tree_connected {
             return;
```

Comment:
This is pre-existing, but this looks very wrong? Why would we never call `bind_to_tree` if unconnected? That sounds like a bug.